vu-registration.js

```diff
@@ -1,143 +1,169 @@
-const fs = require('fs-extra');
-const querystring = require('querystring');
-const cheerio = require('cheerio');
-const FileCookieStore = require('tough-cookie-file-store');
-const jsonic = require('jsonic');
-const sleep = require('sleep-promise');
+const fs = require("fs-extra");
```

Comment:
Does it work? If it does, just run prettier with singleQuote: true and tabWidth: 4 and it should be good

Reply:
Yes, and ok will do